pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -145,6 +145,11 @@ const (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=podgroups,shortName=pg;podgroup-v1beta1
+// +kubebuilder:printcolumn:name="STATUS",type=string,JSONPath=`.status.phase`
```

Comment:
whether pkg/apis/scheduling/types.go need to change as well, please help to eveluate :)

Reply:
`pkg/apis/scheduling/types.go` already change in https://github.com/volcano-sh/apis/pull/76 and https://github.com/volcano-sh/apis/pull/69.

Reply:
@william-wang kindly ping